### PR TITLE
Feature/update to polkadot v0 9 8

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: frame/ddc-metrics-offchain-worker/src/lib.rs [hunk 1]
```diff
@@ -274,7 +274,10 @@ impl<T: Config> Module<T> where <T as frame_system::Config>::AccountId: AsRef<[u
         let s_next_at = StorageValueRef::persistent(b"ddc-metrics-offchain-worker::next-at");
 
         match s_next_at.mutate(|current_next_at| {
-            let current_next_at = current_next_at.unwrap_or(Some(T::BlockNumber::default()));
+            let current_next_at = match current_next_at {
+                Ok(Some(val)) => Some(val),
+                _ => Some(T::BlockNumber::default()),
+            };
```
Comment (MRamanenkau) [inline frame/ddc-metrics-offchain-worker/src/lib.rs:280]: The issue is cause by current_next_at [type changing](https://github.com/paritytech/substrate/pull/9119).
`unwrap_or` behaves differently on `Option` and `Result`. Preveiosly, called on `Option` it has been returning `Some` or default `Some` in parentheses. After refactoring, called on `Result` it never return the default value. Insted, before the fix it has been returning `None` what trigger [else case](https://github.com/Cerebellum-Network/pos-network-node/pull/153/commits/191a12caa3a4c2fd9a3dfb93d868f4647b8fd27f#diff-2cf146c84dbe7f3d47b44a1345d0bed15228bcd3aecb643c6c0abd42f7803a80R299) and the error.

File: bin/node/runtime/src/lib.rs [hunk 2]
```diff
@@ -520,9 +521,14 @@ parameter_types! {
...
 	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
 
-	// fallback: no need to do on-chain phragmen initially.
-	pub const Fallback: pallet_election_provider_multi_phase::FallbackStrategy =
-		pallet_election_provider_multi_phase::FallbackStrategy::Nothing;
+	// signed config
+	pub const SignedMaxSubmissions: u32 = 10;
+	pub const SignedRewardBase: Balance = 1 * DOLLARS;
+	pub const SignedDepositBase: Balance = 1 * DOLLARS;
+	pub const SignedDepositByte: Balance = 1 * CENTS;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:528]: @AndreiNavoichyk 
I can't say how does this affect as. Could you please verify this changes by yourself?

File: primitives/consensus/aura/src/lib.rs [hunk 3]
```diff
@@ -90,7 +90,7 @@ sp_api::decl_runtime_apis! {
 		fn slot_duration() -> SlotDuration;
 
 		// Return the current set of authorities.
-		#[skip_initialize_block]
+
```
Comment (AndreiNavoichyk) [inline primitives/consensus/aura/src/lib.rs]: It makes sense to remove the empty line

File: bin/node/runtime/src/lib.rs (hunk 2 above)
Comment (AndreiNavoichyk) [reply bin/node/runtime/src/lib.rs:528]: @MRamanenkau 
Could you share a PR where Parity introduced these changes?

Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs:528]: [Here it is](https://github.com/paritytech/substrate/pull/7910)

File: primitives/consensus/aura/src/lib.rs (hunk 3 above)
Comment (MRamanenkau) [reply primitives/consensus/aura/src/lib.rs]: Removed

File: bin/node/runtime/src/lib.rs (hunk 2 above)
Comment (AndreiNavoichyk) [reply bin/node/runtime/src/lib.rs:528]: [We need min 4 validators to run staking election](https://substrate.stackexchange.com/questions/630/validator-not-being-onboarded-due-to-election-nofallback-error-possibly)

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**